addons/visibility/functions/fnc_start.sqf

```diff
@@ -0,0 +1,55 @@
+#include "script_component.hpp"
+/*
+ * Author: Brett
+ * Starts the Draw3D eh that indicates when a player can see the curator cursor
+ *
+ * Example:
+ * [] call zen_visibility_fnc_start
+ *
+ * Public: No
+ */
+
+if (GVAR(draw) != -1) exitWith {};
+
+GVAR(draw) = addMissionEventHandler ["Draw3D", {
+    // The the cursor position in the world
+    private _pos = AGLtoASL screenToWorld getMousePosition;
+    private _intersections = lineIntersectsSurfaces [getPosASL curatorCamera, _pos];
+    if ((count _intersections) != 0) then {
+        _pos = ((_intersections select 0) select 0);
+    };
+
+    // check 1.5 above the cursor to prevent a small object on the terrain blocking the view
+    private _posHigh = _pos vectorAdd [0, 0, 1.5];
+    private _draw = false;
+    {
+        if (side _x != sideLogic) then {
+            private _dir = ((_x getRelDir _posHigh) + 90) mod 360;
```

Comment:
`_dir` variable only used in the if condition right below. Can also put both checks in the if statement above?